docs/book/src/cronjob-tutorial/testdata/project/controllers/cronjob_controller.go

```diff
@@ -155,7 +155,7 @@ func (r *CronJobReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 	var mostRecentTime *time.Time // find the last run so we can update the status
 
 	/*
-		We consider a job "finished" if it has a "succeeded" or "failed" condition marked as true.
+		We consider a job "finished" if it has a "Complete" or "Failed" condition marked as true.
```

Comment:
```suggestion
		We consider a job finished if it has a "Complete" or "Failed" condition marked as true.
```

Reply:
@daniel-hutao shows that it is pushing the nit changes.